vnext/Microsoft.ReactNative.Cxx/PropertyBag.h

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#ifndef MICROSOFT_REACTNATIVE_PROPERTYBAG
+#define MICROSOFT_REACTNATIVE_PROPERTYBAG
+
+//!
```

Comment:
nit: I'm not sure anywhere outside of ReactHost uses this comments style. There was a big discussion of what we should use but no clear outcome.

Reply:
This is one of the doxygen commenting style. See: http://www.doxygen.nl/manual/docblocks.html . I have found that it has a nice balance between being VS-friendly and still be usable in doxygen:
- `///` style comments have very special requirements in VS. It expects using XML tags.
- `//` style comments are not doxygen-friendly
- Other start/begin style comment I believe too bulky (take two extra lines) and more difficult to read.

In any case if we ever decide on a common commenting style, I am going to change all of them to follow the common style.